openmetadata-docs/content/deployment/secrets-manager/index.md

```diff
@@ -0,0 +1,87 @@
+---
+
+# Enable Secrets Manager
+
+Secret Manager integrations allow you to use your existing third-party **Key Management Store** (KMS) with OpenMetadata. 
+Your credentials and sensitive information are stored in a tool that you control, and the KMS will mediate between any 
+OpenMetadata internal requirement and sensitive information.
+
+Without a secret manager configured in OpenMetadata, all your sensitive data, any password field of a service connection 
+parameters, bot credentials configuration or DBT configuration of an ingestion pipeline, were stored in MySQL (or 
+Postgres) encrypted.
+
+The following diagram shows how is the process between the OM server and Airflow workflows:
+
+<p/>
+<Image src="/images/deployment/secrets-manager/om-secrets-manager-disabled.png" alt="om-secrets-manager-disabled"/>
+<p/>
+
+As you can see, the `Workflow` consumed by Airflow contains the service information as an `EntityReference`. We use that 
+reference to read the Service information, including its connection details. This information goes from 
+`Database > OM > Airflow`.
+
+When the Secrets Manager is enabled, sensitive information stop being stored in any system from OpenMetadata. Instead, 
+the KMS will act as a mediator, as we can observe in the diagram below:
+
+<p/>
+<Image src="/images/deployment/secrets-manager/om-secrets-manager-enabled.png" alt="om-secrets-manager-enabled"/>
+<p/>
+
+In 0.13 and up, OpenMetadata will communicate through an interface to read/write sensitive information -- removing the 
+need to store sensitive data in OM systems. This new interface works whether users keep using the underlying database of 
+OpenMetadata to store credentials (as it was set up thus far) or any external system such as AWS Secrets Manager or AWS 
+SSM Parameter Store.
+
+In future releases, we will add support for additional Key Management Stores, such as Azure Key Vault or Kubernetes 
+Secrets.
+
+If you’d like to contribute by creating the interface, check the implementation guide, or if you want to see a new one 
+on the supported list, please reach out to us on [Slack](https://slack.open-metadata.org/).
+
+If you are interested in enabling the secrets' manager feature, this is our list of supported Secrets Manager 
+implementations:
+
+- [AWS Secrets Manager](/deployment/secrets-manager/supported-implementations/aws-secrets-manager)
+- [AWS Systems Manager Parameter Store](/deployment/secrets-manager/supported-implementations/aws-ssm-parameter-store)
+
+Things to take into account when enabling the Secrets Manager feature:
+
+1. The migration of all the sensitive data will be done automatically after restarting the OpenMetadata server, which 
+can not be undone for the time being.
+2. Only users with permissions can edit and retrieve the service connections. The connection parameters will be hidden 
+for all other users.
+
+## How it works
+
+There are two types of secrets manager implementations.
+
+### Managed secrets manager
```

Comment:
Should we add a line explaining how to configure a managed vs a non-managed?

Reply:
something like, any implementation can be used as both managed and non managed, the only difference is how the config gets passed

Reply:
It is explained in each implementation section (AWS SSM, AWS secrets manager, ...), and it is almost the same. Here we only explain how it works in each case since it is common for all the implementations.